addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -227,25 +238,27 @@ def _l10n_it_post_invoices_step_2(self, invoices):
             elif state == 'notificaEsito':
                 outcome = response_tree.find('Esito').text
                 if outcome == 'EC01':
-                    to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id}
+                    to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id, 'success': True}
                 else:  # ECO2
                     to_return[invoice] = {'error': _('The invoice was refused by the addressee.'), 'blocking_level': 'error'}
             elif state == 'NotificaDecorrenzaTermini':
-                to_return[invoice] = {'error': _('Expiration of the maximum term for communication of acceptance/refusal'), 'blocking_level': 'error'}
+                to_return[invoice] = {'error': _('Expiration of the maximum term for communication of acceptance/refusal'),
+                                      'success': True}
```

Comment:
Missing `blocking_level`?

Reply:
If the time is over, it should be success.  (If an answer was not given, it is automatically accepted) I adapted and put it in the commit msg as well.

Reply:
It is not what I understand
> Se ci sono dei problemi nella consegna ricevi una notifica di mancata consegna, il SdI ritenta l'operazione per un periodo di 10 giorni al termine del quale, nel caso in cui l'operazione di consegna non sia riuscita, ricevi una notifica di avvenuta trasmissione con impossibilità di consegna.
>
> In questo caso, poichè ci sono dei problemi di ricezione, dovrai contattare l'Amministrazione Destinataria e inviare la fattura con un mezzo diverso dal SdI, dopo un comune accordo.

https://sumup.it/fatture/dizionario/notifica-decorrenza-termini/

Reply:
Dopo l'invio della notifica di decorrenza termini, la fattura PA si considera emessa dal punto di vista fiscale.